cron_tasks.go

```diff
@@ -0,0 +1,27 @@
+package scalingo
+
+import (
+	"gopkg.in/errgo.v1"
+)
+
+type CronTasksService interface {
```

Comment:
This service needs to be added to the `API` interface in client.go. Then regenerate the mocks with `gomock_generator` command (https://github.com/Scalingo/go-utils/tree/master/gomock_generator)